packages/accurapp-scripts/utils/reverse-tunnel.js

```diff
@@ -0,0 +1,45 @@
+const { Socket } = require('net')
+const { Client } = require('ssh2')
+
+function noop() {}
+
+function createClient(config, onReadyCb = noop, onConnectionCb = noop) {
+  const conn = new Client()
+  const errors = []
+
+  conn.on('ready', () => {
+    onReadyCb()
+    conn.forwardIn(config.dstHost, config.dstPort, (err, port) => {
+      if (err) return errors.push(err)
+      conn.emit('forward-in', port)
+    })
+  })
+
+  conn.on('tcp connection', (info, accept, reject) => {
+    let remote
+    const srcSocket = new Socket()
+
+    srcSocket.on('error', err => {
+      errors.push(err)
+      if (remote === undefined) {
+        reject()
+      } else {
+        remote.end()
+      }
+    })
+
+    srcSocket.connect(config.srcPort, config.srcHost, () => {
+      remote = accept()
+      srcSocket.pipe(remote).pipe(srcSocket)
+      if (errors.length === 0) {
+        onConnectionCb(null, conn)
+      } else {
+        onConnectionCb(errors, null)
+      }
+    })
+  })
+  conn.connect(config)
+  return conn
+}
+
+module.exports = { createClient }
```

Comment:
why is this in its own file? you use it only in `tunnel-client`. Please put this in the `tunnel-client` file